Fachportal Padagogik.js

```diff
@@ -0,0 +1,331 @@
+				"issue": "1",
+				"itemID": "article",
+				"libraryCatalog": "Fachportal Pädagogik",
+				"pages": "176-197",
+				"publicationTitle": "Journal for educational research online",
+				"volume": "7",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					},
+					{
+						"title": "Link",
+						"snapshot": false
+					}
+				],
+				"tags": [
+					"Deutschland",
+					"Empirische Untersuchung",
+					"Fragebogen",
+					"Lehrer",
+					"Lernförderung",
+					"Lernmethode",
+					"Mathematikunterricht",
+					"Schuljahr 09",
+					"Selbstgesteuertes Lernen",
+					"Unterrichtsforschung",
+					"Überzeugung"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.fachportal-paedagogik.de/fis_bildung/suche/fis_set_e.html?FId=A18490&mstn=3&ckd=no&mtz=100&facets=y&maxg=5&suche=erweitert&ohneSynonyme=y&feldname1=Freitext&feldinhalt1=MATHEMATIK&bool1=and&nHits=19233&next=A18569,A19036,1065678&prev=A18195,A18196&marker=1",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "10 Jahre international vergleichende Schulleistungsforschung in der Grundschule. Vertiefende Analysen zu IGLU und TIMSS 2001 bis 2011.",
+				"creators": [
+					{
+						"firstName": "Heike ",
```

Comment:
I think these are trimmed in the database so this is fine, but could you check that there are no trailing spaces left after import to be sure? Apart from that, go ahead and squash